code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +154,30 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/M)
+	if(istype(M))
+		var/target = pick(M.organs_by_name)
+		M.apply_damage(rand(200), DAMAGE_BRUTE, target)
+		to_chat(M, SPAN_WARNING("What have you done?! Your [parse_zone(target)] feels like it's being torn from your body! ."))
```

Comment:
Should probably be SPAN_HIGHDANGER since a limb is actively being torn from your body.

Reply:
Change applied